x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/crawler/automatic_crawl_scheduler/automatic_crawl_scheduler_logic.ts

```diff
@@ -80,6 +101,9 @@ export const AutomaticCrawlSchedulerLogic = kea<
         clearCrawlSchedule: () => DEFAULT_VALUES.crawlFrequency,
         setCrawlSchedule: (_, { crawlSchedule: { frequency } }) => frequency,
         setCrawlFrequency: (_, { crawlFrequency }) => crawlFrequency,
+        // TODO Enable this reducer once BE allows
```

Comment:
nitpicking here and not a strong opinion, but is it necessary to leave these commented-out code blocks in here? Feels to me like this should be documented in an issue somewhere rather than in the code.